src/main/java/com/bmc/truesight/meter/plugin/remedy/RemedyTicketsCollector.java

```diff
@@ -129,6 +129,8 @@ public void run() {
                         }//each chunk iteration
                         System.err.println("__________ [Total successful ingestion: " + totalSuccessfulIngestion + ", Total Records from Remedy :" + nMatches.longValue() + ", total iteration " + (iteration - 1) + " ]_____________");
                     }
+                } catch (RemedyLoginFailedException e) {
+                    System.err.println("Remedy Login failed :" + e.getMessage());
```

Comment:
Change the casing of the error message to "Sentence" case

`Remedy login failed`